doc/user_guide/getting_started.rst

```diff
@@ -49,13 +49,14 @@ can be started as follows:
 
 
 
-.. versionadded:: 0.8.1
+.. versionchanged:: 0.8.3
```

Comment:
I'm trying to learn Sphinx here, so I'm trying to figure out what this does. I get why you'd want to add the `versionchanged`, but why remove the `versionadded`? What is the end-result of these tags anyway?